### PR TITLE
fix: Add Jira ignored fields to Jira integration doc

### PR DESCRIPTION
**Pull request by jangjodi**
WOR-2280

Add content about Jira ignored fields to Jira integration doc; specifically, how to find the id or name of the Jira field.

View the corresponding [code change](https://github.com/getsentry/sentry/pull/41445)

### CODE REVIEW

File: src/docs/product/integrations/issue-tracking/jira/index.mdx
Comment (ceorourke) [inline src/docs/product/integrations/issue-tracking/jira/index.mdx:167]: I'd suggest linking to an existing Atlassian doc about this like [this](https://confluence.atlassian.com/jirakb/how-to-find-id-for-custom-field-s-744522503.html) (that was the first one I found, I didn't dig too deeply so there might be a better one) instead of having this section. Then the "The following section will state how to find the field id or name." sentence can be something like "If you need help finding the field's ID, see the Atlassian docs here".

File: src/docs/product/integrations/issue-tracking/jira/index.mdx
Comment (imatwawana) [inline src/docs/product/integrations/issue-tracking/jira/index.mdx:163]:
```suggestion
Ignored fields are Jira fields that are hidden in the issue creation form. To add ignored fields, navigate to **Settings > Integrations** and click "Configure" next to your Jira instance. Then scroll down to “Ignored Fields”.
```

File: src/docs/product/integrations/issue-tracking/jira/index.mdx
Comment (jangjodi) [reply src/docs/product/integrations/issue-tracking/jira/index.mdx:167]: @ceorourke thank you for your feedback! I really appreciate it!

In regards to your first point - I mostly used the [link](https://confluence.atlassian.com/jirakb/how-to-find-id-for-custom-field-s-744522503.html) you mentioned, but I added step 4 (Click the “Select an element” button on the top left of the Inspect module) as I found that I couldn't get the correct ID just from clicking on it (see image below). This may just be an issue on my end though. If you are able to, could you try it and let me know if it worked for you?
6B83215D-2740-4B3E-8945-EE18D13A0D6E

In regards to your second point, yes they are the same thing. I've seen it being called both a ["name"](https://getsentry.atlassian.net/browse/WOR-2280) and ["id"](https://confluence.atlassian.com/jirakb/how-to-find-id-for-custom-field-s-744522503.html). In a discussion with the docs team, I got the feedback to call it "id or name" so that it covers both cases, but if you think one makes more sense I can change it as I can see the confusion.

File: src/docs/product/integrations/issue-tracking/jira/index.mdx
Comment (imatwawana) [inline src/docs/product/integrations/issue-tracking/jira/index.mdx:165]:
```suggestion
Type the ID/name of the Jira field, not the label, into the text input box. The following section explains how to find the field ID/name.
```

File: src/docs/product/integrations/issue-tracking/jira/index.mdx
Comment (imatwawana) [inline src/docs/product/integrations/issue-tracking/jira/index.mdx:167]:
```suggestion
#### Find the ID of the Field
```

File: src/docs/product/integrations/issue-tracking/jira/index.mdx
Comment (imatwawana) [inline src/docs/product/integrations/issue-tracking/jira/index.mdx:177]: Just FYI, you can just number these all as 1. and when it builds, it will increment the numbers correctly.

```suggestion
1. In Jira, access a ticket that has the field that you want the ID/name of.
2. Right click the screen and select “Inspect” or click F12.
3. Click the “Elements” tab of the "Inspect" module.
4. Click the “Select an element” button on the top left of the "Inspect" module.
   ![Select an element button](select-element.png)
5. Click the field that you want the ID/name of.
6. Find the last part of the ‘data-testid’ field, which is the ID/name of the field.
      ![Jira field id or name](jira-field-id-name.png)
```

**Review by imatwawana — COMMENTED** [review]
I'd like to take another quick look at this when you've implemented changes, but overall, it looks good!

File: src/docs/product/integrations/issue-tracking/jira/index.mdx
Comment (ceorourke) [reply src/docs/product/integrations/issue-tracking/jira/index.mdx:167]: Screen Shot 2022-11-16 at 12 44 02 PM

I tried this out and I think it has to do with very specific clicking 😅 If you get the green div it will have the customfield but if you get the blue one it doesn't. The added steps 4 and 5 were confusing until I realized this.

Comment (ceorourke) [reply src/docs/product/integrations/issue-tracking/jira/index.mdx:167]: Yeah I think generally linking to a 3rd party if possible is better than having to maintain our own docs in case things change. 

Regarding the id or name thing, what's confusing to me is that "name" and "label" can be easily conflated, so we're telling them to not use the label but use the name ... as a user I wouldn't know what the difference is, so personally I'd go with ID but I wouldn't block this from going through on this point.

Comment (jangjodi) [reply src/docs/product/integrations/issue-tracking/jira/index.mdx]: Thank you for your response! I made the changes you suggested.

**Comment by jangjodi** [comment]
> I'd like to take another quick look at this when you've implemented changes, but overall, it looks good!

Thank you! I've implemented some changes such that just a link is included instead, but I appreciate your feedback!

**Review by ceorourke — COMMENTED** [review]
LGTM but I'll let @imatwawana have the final say

File: src/docs/product/integrations/issue-tracking/jira/index.mdx
Comment (imatwawana) [inline src/docs/product/integrations/issue-tracking/jira/index.mdx:165]:
```suggestion
Field IDs are different from the labels on the issue creation form. If you need help finding a field's ID, check out the [Atlassian docs](https://confluence.atlassian.com/jirakb/how-to-find-id-for-custom-field-s-744522503.html).
```

File: src/docs/product/integrations/issue-tracking/jira/index.mdx
Comment (imatwawana) [inline src/docs/product/integrations/issue-tracking/jira/index.mdx:163]:
```suggestion
Ignored fields are Jira fields that are hidden in the issue creation form. To add ignored fields, navigate to **Settings > Integrations** and click "Configure" next to your Jira instance. Then scroll down to “Ignored Fields” and enter a comma separated list of the Jira field IDs. 
```

**Review by imatwawana — APPROVED** [review]
Made some last wording tweaks with the new content. Other than that it looks good to me!

**Comment by jangjodi** [comment]
> Made some last wording tweaks with the new content. Other than that it looks good to me!

Thank you, I've made your suggested changes!

**Pull request merged**
